charts/rasa-x/templates/rasa-x-deployment.yaml

```diff
@@ -34,6 +34,7 @@ spec:
       tolerations:
 {{ toYaml .Values.rasax.tolerations | indent 6 }}
       {{- end }}
+      {{- include "initContainer.dbMigration" . |  nindent 6 }}
```

Comment:
Do we even need this here? The container won't be killed as the status endpoint should return a correct status, no?

Reply:
You're right, we don't need it here